audience_manager/form-tracker.rst

```diff
@@ -0,0 +1,101 @@
+.. highlight:: js
+.. default-domain:: js
+
+Audience Manager Form Tracker API
+=================================
+Form Tracker allows to gather data from all forms on your site in one place.
+
+Supported browsers
+------------------
+All modern browsers (Chrome, Firefox, Safari, Edge). Internet Explorer from version 8 and above.
+
+Private information
+-------------------
+Form tracker is trying to automatically detect fields containing private information and exclude them. Following fields
+are always removed from tracked data:
+
+* "password" or "hidden" inputs
+* Credit card number
+* Credit card validation code
+
+Loading snippet
+---------------
+Add following snippet on a page to start using Audience Manager Form Tracker.
```

Comment:
The same issue as for JS API docs - it says "following snippet" and I don't see that snippet as it's much lower and I need to scroll down to find it.

Template tag in tag manager is not mentioned here at all while it is a preferred method of injecting form tracking code, the snippet is an alternative.